api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -3163,6 +3163,76 @@ public CohortDefinition getB2_13(boolean e53) {
     return sqlCohortDefinition;
   }
 
+  /**
+   * <b>RF50</b>: Categoria 13 MG Indicador 13.18 Denominador – Pedido de CV <br>
+   *
+   * <ul>
+   *   <li>incluindo todos os utentes com registo de uma Carga Viral na Ficha Clínica ou Ficha
+   *       Resumo com resultado > 50 cópias durante o período de inclusão (“Data da CV >50” >= “Data
+   *       Início Inclusão” e <= “Data Fim Inclusão”.
+   *   <li>Nota: em caso de existência de mais de um registo de Carga Viral com resultado > =10050
+   *       cópias, deve ser considerado o primeiro registo ocorrido durante o período de inclusão
```

Comment:
Correct the amount of VL copies thta is being evaluated

Reply:
Done